### ISSUE TITLE
No loading indicator when navigating to different pages

### ISSUE THREAD

**Issue by bfirsh** [docker/hub-beta-feedback#176]
When clicking on any internal links, there is no immediate user feedback so when there is high latency (e.g. when you are in Europe!) the page hangs and looks broken.

There should probably be a loading indicator of some kind to immediately show the user that something is happening.

**Comment by ChristopherBiscardi** [comment]
Agreed. We have a temporary stopgap using loading indicators for particularly egregious pages in master.

To give a bit of history, we previously had a "global" loading indicator which was particularly annoying for users. The near-term intent is to move to component-level loading indicators rather than a global page-level loading indicator which aligns better with the continued splitting of the backend into micro-services which each may have varying latency and availability. To move to component-level loading indicators, we needed to remove the global loading indicator.
